akka-docs/rst/project/migration-guide-2.4.x-2.5.x.rst

```diff
@@ -216,6 +216,24 @@ Actor DSL deprecation
 Actor DSL is a rarely used feature and thus will be deprecated and removed.
 Use plain ``system.actorOf`` instead of the DSL to create Actors if you have been using it.
 
+ExtensionKey Deprecation
+------------------------
+
+``ExtensionKey`` is a shortcut for writing :ref:`extending-akka-scala` but extensions created with it
+can not be used from Java and it does in fact not save many lines of code over directly implementing ``ExtensionId``.
```

Comment:
http://www.write.com/writing-guides/general-writing/word-choice/cannot-versus-can-not/ slightly different meaning, I guess cannot is more correct here.